frontend/src/hooks/useFormatTableData.tsx

```diff
@@ -49,7 +49,7 @@ export const formatTableColumns = (
       },
     ];
 
-    Object.entries(tableData.data[0]).forEach((value, index) => {
+    Object.entries(tableData.data.data[0]).forEach((value, index) => {
```

Comment:
how do we know tableData.data.data is always an array

Reply:
Because we built it that way